src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
```

Comment:
This title is a bit long, why not "Importing/Exporting Workflows in the Catalog" ?

Then you can detail in the introduction the multiple options you have available for importing/exporting workflows